File: app/src/main/java/fr/mpau/enums/TypeIntervention.java
package fr.mpau.enums;

/**
 * Author: Jonathan B.
 * Created: 19/01/2018
 */

public enum TypeIntervention {
    SOS("SOS"),
    QUINZE("15");

    private String name = "";

    TypeIntervention(String name) {
        this.name = name;
    }

    public String toString() {
        return name;
    }
}

File: src/main/java/fr/mpau/enums/TypeIntervention.kt
package fr.mpau.enums

/**
 * Enumération des types d'intervention
 *
 * Author: Jonathan
 * Created: 19/01/2018
 * Last Updated: 24/01/2025
 */
enum class TypeIntervention(private val value: String) {

    /**
     * Attributs
     */

    SOS("SOS"),
    QUINZE("15");

    /**
     * Retourne la valeur associée
     *
     * @return String
     */
    fun getValue(): String {
        return value
    }

}